crates/apps/src/lib/node/ledger/shell/finalize_block.rs

```diff
@@ -5080,6 +5086,11 @@ mod test_finalize_block {
     /// Test that updating the ethereum bridge params via governance works.
     #[tokio::test]
     async fn test_eth_bridge_param_updates() {
+        if !is_bridge_comptime_enabled() {
```

Comment:
could we keep the bridge enabled for tests? i.e. `#[cfg(any(feature = "namada-eth-bridge", test))]`

Reply:
I was trying to avoid sprinkling `cfg` flags all over the place, instead centralizing them under the predicate `is_bridge_comptime_enabled()`. Either way, most of these tests would simply not work if they ran with `#[cfg(not(feature = "namada-eth-bridge"))]` verifying (i.e. with the Bridge disabled by a compile time flag). The things we should test are that protocol txs get rejected by VPs/the mempool, which we already do with the changes in this PR.

Reply:
Ah, I don't want to nitpick on this, but I think if we enable the feature in the apps' `dev-deps` for `namada`, we would only have to use `#[cfg(any(feature = "namada-eth-bridge", test))]` (and its negation) in the apps crate. Then
- the rest could stay the same as is
- we can remove `is_bridge_comptime_enabled` altogether as the tests would always run
- we wouldn't have to run unit tests 2 times in CI